scripts/lua/if_stats.lua

```diff
@@ -1960,6 +2014,39 @@ print [[</form>
    var hash = window.location.hash;
    $('#filterPageTabPanel a[href="' + hash + '"]').tab('show');
    /*** End Page Tab State ***/
+   $(function() {      
```

Comment:
see if you can generalize this method to use it both here and for the preferences page